### PR TITLE
Stacktraces on kernel panic

### PR DESCRIPTION
**Pull request by ry-diffusion**
NOTICE: I don't know if this is the _correct_ way to do this.

### CODE REVIEW

File: Kernel/Kernel/Stack.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    __asm__ volatile("mov %%rbp, %0" : "=r"(rbp));
+
+    for (uint64_t frame = 0; rbp && frame < DUMP_STACK_ENTRIES_MAX; ++frame) {
+        kprintf(" [%p]\n", rbp);
+        rbp += sizeof(void *);
+    }
+}
+
+#elif defined(__i386__)
```
Comment (RedsonBr140) [inline Kernel/Kernel/Stack.c]: We can drop the i386 code completely as I don't plan on porting blobOS to this architecture.

The if macro is a good idea tho, it'll be helpful when we add support for other architecture :)

File: Kernel/Arch/x86_64-pc/System/GDTFarJump.asm [hunk 2]
```diff
@@ -1,5 +1,6 @@
 [bits 64]
 [global reloadSegments]
+[global set_up_stack]
```
Comment (RedsonBr140) [inline Kernel/Arch/x86_64-pc/System/GDTFarJump.asm]: You should probably remove this line as there's no function called "set_up_stack" and no place where we use it

**Review by RedsonBr140 — CHANGES_REQUESTED** [review]
It looks good for now. Later on, I will unwind the stack so that we can get function names, not just their addresses. There are just little things I'd like you to change.

File: Kernel/Arch/x86_64-pc/System/GDTFarJump.asm (hunk 2 above)
Comment (ry-diffusion) [reply Kernel/Arch/x86_64-pc/System/GDTFarJump.asm]: Yeah, It was for some 32 bit things. I forgot to remove It, thanks

File: Kernel/Kernel/Stack.c (hunk 1 above)
Comment (ry-diffusion) [reply Kernel/Kernel/Stack.c]: I see. That code hasn't been tested so it's safe to remove

**Review by RedsonBr140 — APPROVED** [review]

**Pull request merged**